src/lib/SX127xDriver/SX127x.h

```diff
@@ -44,6 +44,7 @@ class SX127xDriver
     SX127x_RadioOPmodes currOpmode = SX127x_OPMODE_SLEEP;
     uint8_t currPWR = 0b0000;
     SX127x_ModulationModes ModFSKorLoRa = SX127x_OPMODE_LORA;
+    bool IQinverted = false;
```

Comment:
Why this is saved? Is that needed later?
Please remove all unnecessary parameters if not used.

Reply:
Had the idea that maybe Telemetry could sent with inverse LQ to reduce interference, so I wanted to know state. I think it won't make a difference. I'll remove the variable.